services/continuous_querier/service.go

```diff
@@ -353,6 +355,7 @@ func (s *Service) runContinuousQueryAndWriteResult(cq *ContinuousQuery) error {
 	// Execute the SELECT.
 	ch := s.QueryExecutor.ExecuteQuery(q, influxql.ExecutionOptions{
 		Database: cq.Database,
+		Quiet:    !s.logQueries,
```

Comment:
This would require a much larger change since false is the default. We would have to change every time a set of execution options were created and any future times we use this code.